tests/e2e/detect_error_logs_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 The etcd Authors
```

Comment:
As for file name `tests/e2e/detect_error_logs_test.go`, don't think we need to be so specific. We don't plan to as many test cases for detecting error logs. `tests/e2e/logging_test.go` should suffice.